pom.xml

```diff
@@ -112,6 +113,31 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
ℹ️ I've tested this with resources with Kotlin and Java.

There are few gochas to keep in mind:
1. If you just run `jar:jar` -> the jar is generated by different plugin and unusable
2. If you just run `assembly:single` -> the correct jar will be generated, but will be empty
3. If you run `install` -> both jars will be generated, and they will contain everything. But candidate will need to run the "correct" of the 2 jar files. 🤷

Reply:
We could just disable jar or use fatjar instead

Reply:
Disabled it.

Reply:
Undid the disable, as it was causing `maven install` to fail (even if it produced only the valid JAR before failing.